docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+---
+title: App Management Overview
+---
+
+{% capture overview %}
+
+*Kubectl* supports several different approaches for creating and managing
+applications in a Kubernetes cluster.  This document provides an overview
```

Comment:
"applications" is an imprecise term. In the limit an Application is going to be all sorts of config that may stretch outside of k8s.  And Management is more than just configuring k8s.  I'd say that these are "Kubernetes Config Management" techniques.  But "Config Management" is a heavily loaded term.